### PR TITLE
Roaming fix

### PR DESCRIPTION
**Pull request by JoseVOliveria**
Hi @herlesupreeth

First of all thanks for the repository and your excellent work! 
With the help of @o2valente we updated your branch roaming and its config files. This way by deploying one single compose file the core can serve both as home and visited. 

The main changes were done to the following files :
- .env - it were added the variables of the Roaming MNC (MNC_R) and MCC (MCC_R) and the IP of the Remote host where the Roaming core is deployed (it was used in the sepp.yaml).
-  pcf.yaml - it was added the policies of  connected networks
-  amf.yaml - it was added the access control for the PLMNs of the connected networks
- sepp.yaml - it was added the remote host

It were added FQDN´s in the ausf, udm, and nrf . We tried using your dns container but the NF's only connected to the NRF container when we added the alias in the compose file.

We ran the sa-vonr-deploy.yaml in two separate hosts where it were changed the values of MNC ,MCC, MCC_R, MNC_R, Docker_HOST_IP, DOCKER_HOST_IP_R AND UPF_ADVERTISE_IP. 
The first machine was configured with the PLMN 00101  and the second with the PLMN 00103. We then had 2 USRP B200 connected to each core as gNB.

We also added PCF policies for both Internet and IMS, because we tested VoNR with Roaming. I think leaving it would be of use, even if commented as we didnt find any examples for multiple APNs in the yamls.

With a Xiaomi Red Note 9T and a programmable SIM card with a PLMN 00101 we were able to connect to both the home network (00101) and visited network (00102) through the USRP B200 with the PLMN 00102. We were also able to connect directly to the 00102 core with a sim programmed with a 00102 PLMN.

Hope the PR is of use to you. Let me know if you have any doubts.

### CODE REVIEW

**Comment by herlesupreeth** [comment]
Hey, thank you very much for the PR. I am on vacation currently will review this once I am back

File: ims_base/modules.lst [hunk 1]
```diff
@@ -7,7 +7,7 @@ modules_dirs:=modules
 cfg_group_include=
 
 # the list of extra modules to compile
-include_modules= cdp cdp_avp db_mysql dialplan enum json http_client ims_auth ims_charging ims_dialog ims_diameter_server ims_icscf ims_ipsec_pcscf ims_isc ims_ocs ims_qos ims_registrar_pcscf ims_registrar_scscf ims_usrloc_pcscf ims_usrloc_scscf outbound presence presence_conference presence_dialoginfo presence_mwi presence_profile presence_reginfo presence_xml pua pua_bla pua_dialoginfo pua_reginfo pua_rpc pua_usrloc pua_xmpp sctp tls utils xcap_client xcap_server xmlops xmlrpc
+include_modules= cdp cdp_avp db_mysql dialplan enum http_async_client http_client ims_auth ims_charging ims_dialog ims_diameter_server ims_icscf ims_ipsec_pcscf ims_isc ims_ocs ims_qos ims_registrar_pcscf ims_registrar_scscf ims_usrloc_pcscf ims_usrloc_scscf jansson json nghttp2 outbound presence presence_conference presence_dialoginfo presence_mwi presence_profile presence_reginfo presence_xml pua pua_bla pua_dialoginfo pua_reginfo pua_rpc pua_usrloc pua_xmpp sctp tls utils uuid xcap_client xcap_server xmlops xmlrpc
```
Comment (herlesupreeth) [inline ims_base/modules.lst:10]: I dont understand why IMS related changes are present in this PR related to roaming

File: osmohlr/Dockerfile [hunk 2]
```diff
@@ -39,4 +39,4 @@ RUN wget $OSMOCOM_REPO/Release.key && \
     apt-get update && apt-get install -f && apt-get --install-suggests -y install osmo-hlr
 
 CMD /mnt/osmohlr/osmohlr_init.sh && \
-    cd /mnt/osmohlr && /usr/bin/osmo-hlr -c /etc/osmocom/osmo-hlr.cfg
+    cd /mnt/osmohlr && /usr/bin/osmo-hlr -c /etc/osmocom/osmo-hlr.cfg --db-upgrade
```
Comment (herlesupreeth) [inline osmohlr/Dockerfile:42]: I dont understand why this change is present in the PR related to roaming

File: osmohlr/osmo-hlr.cfg [hunk 3]
```diff
@@ -16,5 +16,6 @@ line vty
 ctrl
  bind 127.0.0.1
 hlr
+ subscriber-create-on-demand 5 cs+ps
```
Comment (herlesupreeth) [inline osmohlr/osmo-hlr.cfg:19]: same as above

File: osmomsc/osmo-msc.cfg [hunk 4]
```diff
@@ -1,11 +1,17 @@
 !
 ! OsmoMSC configuration saved from vty
 !
+log stderr
```
Comment (herlesupreeth) [inline osmomsc/osmo-msc.cfg:4]: same as above

**Review by herlesupreeth — COMMENTED** [review]
Hey!! I see that there are changes unrelated to roaming. Can you please remove them and keep the PR clean by addressing a single issue/feature?

**Comment by JoseVOliveria** [comment]
Hello @herlesupreeth,
I have done some cleanup to the previous commit leaving only the roaming feature deployed in the sa-deploy.yaml file.
Sorry for the inconvenience, I am doing some work with both VoNR and Roaming that's why there were both features.

Thank you
José Oliveira
